src/components/ha-combo-box.ts

```diff
@@ -304,6 +305,17 @@ export class HaComboBox extends LitElement {
     }
   }
 
+  private _handleRTL(overlay: HTMLElement) {
+    overlay
+      .querySelectorAll("vaadin-combo-box-scroller > vaadin-combo-box-item")
+      .forEach((item) => {
+        const elem = item as HTMLElement;
+        if (elem.dir === "rtl") {
+          elem.style.padding = "0px";
```

Comment:
Can we add this to https://github.com/home-assistant/frontend/pull/15233/files#diff-32fb6be2475d4e810c29381a10e4c784eb279630c924063afafe53fd9a4d43dfL22-L23 `registerStyles` for "vaadin-combo-box-item"?

Reply:
I started with that. Those styles are nowhere to be seen. Seems like it's broken?

Reply:
I see them? 
![CleanShot 2023-01-30 at 12 47 00](https://user-images.githubusercontent.com/5662298/215468646-26d53c18-ccbc-4c40-8d54-1036e8dc5213.png)

Reply:
Also the padding on that element already is set to `0` right? What are we trying to fix exactly?

Reply:
I added !important since it wasn't being applied in RTL. Don't ask me why.